### PR TITLE
docs(telemetry): Direct readers to Kedro documentation for further information on telemetry

### PR DESCRIPTION
**Pull request by astrojuanlu**
## Description

https://github.com/kedro-org/kedro-plugins/issues/508, https://github.com/kedro-org/kedro/pull/3603

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the relevant `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by DimedS — APPROVED** [review]
Thank you, @astrojuanlu . I found the examples in that table to be illustrative and valuable. Perhaps we should consider incorporating some of them into https://docs.kedro.org/en/stable/configuration/telemetry.html, such as `kedro run --pipeline=ds --env=test `becoming` kedro run --pipeline ***** --env *****`.

**Comment by astrojuanlu** [comment]
Good point @DimedS, do you want to send a quick PR? Or I can do it myself

File: kedro-telemetry/README.md [hunk 1]
```diff
@@ -5,39 +5,20 @@
...
-## Introduction
+## What is kedro-telemetry?
 
-To find out how Kedro's features are used, the [Kedro team](https://github.com/kedro-org/kedro) have created an opt-in Kedro-Telemetry [plugin](https://docs.kedro.org/en/stable/extend_kedro/plugins.html) to gather anonymised (before being sent across the internet) and aggregated usage analytics.
-
-The data it gathers will help drive future development of Kedro because we can understand how the product is being used.
+Kedro-Telemetry is a [plugin](https://docs.kedro.org/en/stable/extend_kedro/plugins.html)
+that gathers anonymised and aggregated usage analytics
+to help [the Kedro team](https://docs.kedro.org/en/stable/contribution/technical_steering_committee.html)
+improve the software.
```
Comment (stichbury) [inline kedro-telemetry/README.md:13]:
```suggestion
understand how Kedro is used and prioritise improvements to the product accordingly.
```

File: kedro-telemetry/README.md [hunk 2]
```diff
@@ -5,39 +5,20 @@
...
-
-## What about my personal data?
-
-The Kedro Project’s telemetry has been reviewed and approved under the [Telemetry Data Collection and Usage Policy](https://lfprojects.org/policies/telemetry-data-policy/) of LF Projects, LLC.
-
-
-## What data is being collected?
-
-We process your hashed computer username, which is pseudonymized information that indirectly relates to you personally. We collect the following project-related and environmental information, relying on your consent to do so:
+Read [our Telemetry collection documentation](https://docs.kedro.org/en/stable/configuration/telemetry.html)
```
Comment (stichbury) [inline kedro-telemetry/README.md:17]:
```suggestion
Read [our Telemetry documentation](https://docs.kedro.org/en/stable/configuration/telemetry.html)
```

File: kedro-telemetry/README.md [hunk 3]
```diff
@@ -5,39 +5,20 @@
...
+for further information on the intent of the data collection and what data is collected.
```
Comment (stichbury) [inline kedro-telemetry/README.md:18]: This doesn't answer "why?" which is itself answered in the section above that says it's to improve Kedro. I'd rephrase the question along the lines "What data is collected?"

**Review by stichbury — APPROVED** [review]
Minor feedback suggestions.

File: kedro-telemetry/README.md (hunk 3 above)
Comment (astrojuanlu) [reply kedro-telemetry/README.md:18]: Good point, amending
